pkg/agent/flowexporter/exporter/util.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 Antrea Authors
```

Comment:
you may need to update the copyright notice for new files to use 2021 instead of 2020. Otherwise I don't think CI should pass?

Reply:
Update. Thanks!